packages/_flutter_web_build_script/lib/build_script.dart

```diff
@@ -261,6 +261,7 @@ class FlutterWebTestBootstrapBuilder implements Builder {
 
     if (metadata.testOn.evaluate(SuitePlatform(Runtime.chrome))) {
       await buildStep.writeAsString(id.addExtension('.browser_test.dart'), '''
+// @dart = 2.8
```

Comment:
There's usually a blank line around these, but I guess it really doesn't matter here.